project/LibraryVersions.scala

```diff
@@ -2,8 +2,9 @@
 object LibraryVersions {
   val circeVersion = "0.12.1"
   val awsClientVersion = "1.11.827"
-  val catsVersion = "1.4.0"
-  val jacksonVersion = "2.12.0"
+  val catsVersion = "2.1.1"
+  val jacksonVersion = "2.10.5"
+  val jacksonDatabindVersion = "2.10.5.1"
```

Comment:
scala jackson module now throws an error if the jackson databind version is not the same minor version, so downgrading.
I do think that we need urgently to get onto play 2.8 and AWS client v2 asap, as this problem will be impossible to get out of soon, as jackson say this version is no longer supported and they will not release any more updates.